File: Spring/Chapter7/MyBatisSpringJavaInit/src/main/java/pers/oneToMany/entity/UserWithOrders.java
package pers.oneToMany.entity;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.List;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class UserWithOrders {
    private Integer id;
    private String name;
    private Short age;
    private List<Orders> ordersList;
}

File: Spring/Chapter7/MyBatisSpringKotlinInit/src/main/kotlin/pers/oneToMany/entity/UserWithOrders.kt
package pers.oneToMany.entity

open class UserWithOrders {
    var id = 0
    var name = ""
    var age:Short = 0
    var ordersList:List<Orders> = mutableListOf()
    override fun toString() = "UserWithOrders(id:$id,name:$name,age:$age,ordersList:$ordersList)"
}